src/css/components/_footer.scss

```diff
@@ -52,8 +55,10 @@
   width: 90px;
   &__link {
     color: $color-typography-footer-icon;
+    @apply dark:text-typography-title !important;
 
     img {
+      @apply dark:scylla-icon-svg;
```

Comment:
idk if is the point of change about the footer, but, the gif show scylla logo dark, wouldn't it be better put the scylla logo white? Print for example
![image](https://github.com/user-attachments/assets/91f6b016-50fd-48fb-8fa3-d63c99ae4838)

Reply:
I missed this one. I'll fix it later, thanks!